liteflow-labs/starter-kit
# Implement token media size

### Project organization

- Dependant on https://github.com/liteflow-labs/starter-kit/pull/159

### Description

Properly load the image right size by adding/implementing the `sizes` props of the Next Image component.

When using the Next Image component using `layout="fill"`, if no width/height or sizes are specified it default to the maximum of 3840px.

### TODO for future PR:
- There are still a few places that use the Next Image component without setting width/height or sizes, they will need to be updated as well: https://github.com/liteflow-labs/starter-kit/pull/166

### How to test

- Open the website on the dev branch (https://liteflow-nft-test-dev-polygon-mumbai.vercel.app/)
- Open your network console, filter for `img`
- Navigate to the homepage or any page with nft's image
- Refresh
- Look at the requests of the nft's image: check the param `w`
- Notice it's always `3840` (maximum size)
- Now open the same page on this branch (https://nft-test-polygon-mumbai-git-feature-token-media-size-liteflow.vercel.app/)
- Notice the param `w` is a lot smaller 🎊 

### Checklist
